src/bin/common.rs

```diff
@@ -369,6 +370,14 @@ pub fn parse_cli() -> Result<CliOptions, CliError> {
         .help("Overwrite output file.")
         .short("y")
     )
+    .arg(
+      Arg::with_name("SLOTS")
```

Comment:
Perhaps this argument should be hidden for stable builds. I'm not sure how to ergonomically exclude it with this builder syntax.

Reply:
I'll get that sorted.